libs/@guardian/identity-auth/src/@types/OAuth.ts

```diff
@@ -13,18 +21,17 @@ import type { AccessToken, CustomClaims, IDToken } from './Token';
  * @param autoRenew - Whether to automatically renew the tokens, defaults to `true`
  * @param renewGracePeriod - The time in seconds before the access token expires to renew the token, defaults to 60 seconds
  * @param maxClockSkew - The maximum time drift in seconds between the client and server, defaults to 300 seconds (5 minutes), based on the default maximum tolerance of the Kerberos protocol
+ * @param idCookieSessionRefresh - If switched on (set to `true`, ), will refresh the okta session and identity cookie (GU_U), if required (once every 30 days), should be based on the id-cookie-refresh (idCookieRefresh) switch if being supplied by theguardian.com, defaults to `false`
  */
 export interface IdentityAuthOptions {
 	clientId: string;
-	issuer: `https://profile.${
-		| 'theguardian'
-		| 'code.dev-theguardian'
-		| 'thegulocal'}.com/oauth2/${string}`;
+	issuer: `${ProfileUrl}/${string}`;
```

Comment:
Did you lose the `/oauth2` part of the string?

Reply:
Good spot! Yes I did.